WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanViewController.swift

```diff
@@ -25,8 +25,15 @@ class JetpackScanViewController: UIViewController, JetpackScanView {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        self.title = NSLocalizedString("Scan", comment: "Title of the view")
+
         configureTableView()
         coordinator.viewDidLoad()
+
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "History",
```

Comment:
👀 "History" button should be localized

Reply:
NooOooO!!!! Haha, done in e42ca2ca068b4651c256fbdf010b2d16fca062d3